File: FizzBuzz.js
function fizzBuzz(n) {
    // Loop from 1 to n (inclusive)
    for (let i = 1; i <= n; i++) {
        // Check if the number is divisible by both 3 and 5
        if (i % 3 === 0 && i % 5 === 0) {
            console.log("FizzBuzz"); // Print "FizzBuzz" for multiples of both 3 and 5
        // Check if the number is divisible by 3
        } else if (i % 3 === 0) {
            console.log("Fizz"); // Print "Fizz" for multiples of 3
        // Check if the number is divisible by 5
        } else if (i % 5 === 0) {
            console.log("Buzz"); // Print "Buzz" for multiples of 5
        } else {
            console.log(i); // Print the number if it's not divisible by 3 or 5
        }
    }
}

// Example usage: print numbers from 1 to 15 with FizzBuzz substitutions
fizzBuzz(15);

File: FizzBuzz.py
def fizz_buzz(n):
    # Loop from 1 to n (inclusive)
    for i in range(1, n + 1):
        # Check if the number is divisible by both 3 and 5
        if i % 3 == 0 and i % 5 == 0:
            print("FizzBuzz")  # Print "FizzBuzz" for multiples of both 3 and 5
        # Check if the number is divisible by 3
        elif i % 3 == 0:
            print("Fizz")  # Print "Fizz" for multiples of 3
        # Check if the number is divisible by 5
        elif i % 5 == 0:
            print("Buzz")  # Print "Buzz" for multiples of 5
        else:
            print(i)  # Print the number if it's not divisible by 3 or 5

# Example usage
fizz_buzz(15)
